### PR TITLE
New example Poisson DG

### PR DESCRIPTION
**Pull request by ghislainb**

### CODE REVIEW

File: src/example/poisson_dg/Project.toml
Comment (bmxam) [inline src/example/poisson_dg/Project.toml:1]: can you specify the Bcube version number in a [compat] block please?

File: src/example/poisson_dg/poisson_dg.jl [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const f = PhysicalFunction(x -> -2 - 12 * x[1])
+const g = uₐ
+
+avg(u) = 0.5 * (side⁺(u) + side⁻(u))
+
+function main()
+
+    # Build mesh
+    meshParam = (nx = n + 1, ny = n + 1, lx = Lx, ly = Lx, xc = 0.0, yc = 0.0)
+    tmp_path = "tmp.msh"
```
Comment (bmxam) [inline src/example/poisson_dg/poisson_dg.jl:30]: `tmp_path = joinpath(mktemp(), "mesh.msh")`

**Review by bmxam — APPROVED** [review]
Great : just minor remarks

File: src/example/poisson_dg/poisson_dg.jl (hunk 1 above)
Comment (ghislainb) [reply src/example/poisson_dg/poisson_dg.jl]: done

File: src/example/poisson_dg/poisson_dg.jl [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    l_Ω(v) = ∫(v * f)dΩ
+
+    function a_Γ(u, v)
+        ∫(
+            -jump(v, nΓ) ⋅ avg(∇(u)) - avg(∇(v)) ⋅ jump(u, nΓ) +
+            γ / h * jump(v, nΓ) ⋅ jump(u, nΓ),
+        )dΓ
+    end
+
+    fa_Γb(u, ∇u, v, ∇v, n) = -v * (∇u ⋅ n) - (∇v ⋅ n) * u + (γ / h) * v * u
```
[the remark this answers is not in the document]
Comment (ghislainb) [reply src/example/poisson_dg/poisson_dg.jl:59]: a link to the Gridap tutorial was added

**Pull request merged**